nox/virtualenv.py

```diff
@@ -349,17 +350,21 @@ def _clean_location(self) -> bool:
 
     def _check_reused_environment_type(self) -> bool:
         """Check if reused environment type is the same."""
-        path = os.path.join(self.location, "pyvenv.cfg")
-        if not os.path.isfile(path):
+        try:
+            with open(os.path.join(self.location, "pyvenv.cfg")) as fp:
+                parts = (x.partition("=") for x in fp if "=" in x)
+                config = {k.strip(): v.strip() for k, _, v in parts}
+            if "gourgeist" in config:
```

Comment:
```suggestion
            if "uv" in config or "gourgeist" in config:
```
Being changed to "uv" in https://github.com/astral-sh/uv/pull/1852.